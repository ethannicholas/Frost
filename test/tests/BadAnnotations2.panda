@class
def x := 10

@class
method main() {
}