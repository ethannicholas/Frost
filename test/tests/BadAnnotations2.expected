BadAnnotations2.frost:2:5: error: '@class' annotation may not be used outside of a class
BadAnnotations2.frost:5:1: error: '@class' annotation may not be used outside of a class
2 errors
